src/globus_sdk/globus_app/app.py

```diff
@@ -283,7 +271,7 @@ def login_required(self) -> bool:
 
     def logout(self) -> None:
         """
-        Logout an auth entity from the app.
+        Logout an auth entity out from the app.
```

Comment:
I'm not sure why I'm only seeing this now, but I think either this needs to revert or it needs to change to
`Log ... out from the app`.

Otherwise, this is saying "logout out of the app".

Trying to find my "best liked" phrasing without spending more than 30s on this, I come up with...
```
Log the current user or client out of the app.
```

Reply:
Updated to use "a user or client" in this method and login